python/ray/serve/_private/replica_scheduler/replica_wrapper.py

```diff
@@ -105,6 +109,7 @@ async def send_request_python(
             return ActorReplicaResult(obj_ref_gen, pr.metadata), queue_len_info
         except asyncio.CancelledError as e:
             # HTTP client disconnected or request was explicitly canceled.
+            logger.info("Cancelling replica result.")
```

Comment:
this log will show to users and is likely confusing (replica result is a purely internal concept)

Suggest to make it more clear from a behavioral standpoint:
```suggestion
            logger.info("Cancelling request that has already been assigned to a replica.")
```